net.imaginaryinfinity.Squiid.json

```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "net.imaginaryinfinity.Squiid",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "squiid",
+    "finish-args": [
+        "--share=network"
```

Comment:
this display nothing?

Reply:
It doesn't display anything graphically if that's what you're asking. It uses IPC internally which requires this network flag for some reason. I need to close and reopen the PR because I forgot my generated_sources.json file and it won't let me update it.